### ISSUE TITLE
Make an adminisrative script to create REST API token for SDS service

### ISSUE THREAD

**Issue by hjhsalo** [secureb2share/secureb2share-b2share#4]
TL;DR

Python script should be implemented that:
1. Creates user account for SDS service
2. Creates a REST API token for user account of SDS service

This script will be run with `b2share shell`

In order to interact with B2SHARE REST API, SDS service needs to have an REST API token from B2SHARE.

Since REST API tokens in B2SHARE are user account specific, SDS needs to have an user account at B2SHARE.

Ideally user account created for SDS service, cannot be used to login into B2SHARE; it should only be possible to use the REST API token created for this user account.

Currently B2SHARE doesn't allow creating REST API tokens for an user account with first logging in with that user account. (even superadministrator account cannot create REST API tokens for other accounts)

Since we don't want to enable login with SDS account, creating token for SDS account can only be done via B2SHARE shell:
- https://github.com/EUDAT-Training/B2SHARE-Training/blob/master/deploy/A_b2share_Tool_Reference.md#shell
- http://flask.pocoo.org/docs/1.0/cli/#open-a-shell
